logstash-core/spec/logstash/settings/port_range_spec.rb

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+#
+require "logstash/settings"
+require "spec_helper"
+
+describe LogStash::Setting::PortRange do
+
+  context "When the value is a Fixnum" do
+    subject { LogStash::Setting::PortRange.new("mynewtest", 9000) }
+
+    it "coerces the value in a range" do
+      expect { subject }.not_to raise_error
+    end
+
+    it "returns a range" do
+      expect(subject.value).to eq(9000..9000)
+    end
+
+    it "can update the range" do
+      subject.set(10000)
+      expect(subject.value).to eq(10000..10000)
+    end
+  end
+
+  context "When the value is a string" do
+    subject { LogStash::Setting::PortRange.new("mynewtest", "9000-10000") }
```

Comment:
The default range is not set here, but in https://github.com/elastic/logstash/pull/5774/files/b7509e80ea0a4a4b8eff35d15f46a22d62e93c97#diff-b803768250328614e95dee4f5b27af18R4, this spec file is only for testing the `PortRange` class.

To correctly test the default port it require a new assertion in`qa/acceptance`, will check on hard it is to add it.